src/ontology/edits/oeo-shared.omn

```diff
@@ -10,8 +10,8 @@ Prefix: xsd: <http://www.w3.org/2001/XMLSchema#>
 
 Ontology: <http://openenergy-platform.org/ontology/oeo/oeo-shared/>
 <http://openenergy-platform.org/ontology/oeo/dev/oeo-shared.omn>
+Import: <http://openenergy-platform.org/ontology/oeo/dev/oeo-import-edits.owl>
 Import: <http://openenergy-platform.org/ontology/oeo/dev/imports/iao-extracted.owl>
-Import: <http://openenergy-platform.org/ontology/oeo/dev/imports/omo-extracted.owl>
```

Comment:
If omo-extracted.owl is not imported any more we should not only remove the import statement here, but also delete this file completely. Else we would leave an orphanes file without any use.